releasenotes/notes/ad-ignore-stopped-containers-ee856d1e3df0bf7d.yaml

```diff
@@ -0,0 +1,7 @@
+---
+enhancements:
+  - |
+    Introduce a `container_exclude_stopped_after` configuration option to allow
```

Comment:
I'm finding this name a bit confusing -- it sounds like it would ignore things stopped after a certain time, when in fact it does the opposite.  Perhaps `container_exclude_stopped_age`?

Reply:
Hmm, re-reading this a few days after writing it and I agree with you. I'll adopt your suggestion, thanks!